src/njoy21/input/ACER/Card7/Thin2.hpp

```diff
@@ -0,0 +1,12 @@
+struct Thin2 {
+  using Value_t = double;
+  static std::string name() { return "thin2"; }
+
+  static std::string description(){
+    return 
+        "The thin2 argument specifies the energy above which to use all\n"
+        "energies (eV)---i.e., no thinning above this energy.\n";
+  }
+  static Value_t defaultValue(){ return 0.0; }
+  static bool verify( const Value_t& e2 ){ return e2 >= 0.0; }
```

Comment:
As I look closer at the code, I don't think thinning is ever performed—regardless of what the user inputs. 

From acer.f90 starting at line 314
```fortran
z(1)=0
z(2)=0
z(3)=0
read(nsysi,*) (z(i),i=1,3)
thin(1)=abs(z(1))
thin(2)=z(2)
thin(3)=z(3)
thin(4)=0
if (z(1).ge.zero.and.thin(2).gt.thin(1)) thin(4)=1
if (z(1).lt.zero) thin(4)=2
if (thin(4).eq.one) iskf=nint(thin(3))
if (thin(4).eq.one) write(nsyso,'(&
  &'' energy at which to start thinning .... '',1p,e10.2/&
  &'' energy at which to stop thinning ..... '',e10.2/&
  &'' skip factor .......................... '',i10)')&
  thin(1),thin(2),iskf
if (thin(4).eq.two) iwtt=nint(thin(1))
if (thin(4).eq.two) npts=nint(thin(2))
if (thin(4).eq.two) write(nsyso,'(&
  &'' thinning wt option (1 flat, 2 1/e) ... '',i10/&
  &'' target no. of points ................. '',i10/&
  &'' reference sigma zero ................. '',1p,e10.2)')&
  iwtt,npts,thin(3)
```

Nowhere else in `acer.f90` are the variables: `thin`, `iwtt`, `npts`, or `iskf` used. I think the thinning portion must have been removed sometime; I'm guessing when switching to NJOY2012.